docs/architecture/api-credentials-flows.md

```diff
@@ -0,0 +1,18 @@
+# Overview
+
+This document describes how [Application](/docs/terminology.md#Application) provides and updates credentials for [API Definition](/docs/terminology.md#Application-API-Definiton) per [Runtime](/docs/terminology.md#Runtime).
+
+## Details
+This diagram illustrates the API credentials flow in details: 
+
+>**NOTE:** The Application provides [Webhook API](/docs/terminology.md#Application-Webhook-API) where [Management Plane](/docs/terminology.md#Management-Plane) will ask for new credentials for given API Definition.
+
+>**NOTE:** The Application or [Administrator](/docs/terminology.md#Administrator) can exchange credentials for registered APIs at any time.
+
+![Application Webhook](./assets/api-credentials-flow.svg)
```

Comment:
- describe Administrator actor (below the blue icon)
- please add more space between component heading and the component block
- I would recommend using simple arrows instead of "many to one" relation arrows. Maybe it would be better to write Application x n and Runtime x n, similarly to my diagrams?

Reply:
this block is ~2pxs off
![Screen Shot 2019-06-03 at 15 42 56](https://user-images.githubusercontent.com/7155799/58806484-8cbb5300-8616-11e9-96e6-e1513a86d607.png)